chain/client/src/client.rs

```diff
@@ -2257,6 +2261,9 @@ impl Client {
                     .insert(bp.public_key().clone());
             }
         }
+        for v in self.epoch_manager.get_epoch_all_validators(&tip.epoch_id)? {
+            account_keys.entry(v.account_id().clone()).or_default().insert(v.public_key().clone());
+        }
```

Comment:
Why not also do it for the next epoch?

Reply:
I guess I am wary of increasing the size of the T1 set more aggressively than we need to. A few missed endorsements at the start of the new epoch while new T1 connections are established should be a relatively minor issue. At the same time, if we think the CV set is generally quite stable and this doesn't really increase the connection count, I am fine with including next epoch CVs.

Reply:
TBH I would just make it consistent with the BPs and CPs to avoid any confusion around this in the future. Yes, I think we can safely assume this set is quite stable over time.